### ISSUE TITLE
unbound module

### ISSUE THREAD

**Issue by OlivierJM** [benjamminj/reason-react-tutorial#1]
Nice and awesome tutorial.

I think this is a good path for me to learning reason-react, I keep getting an error of unbound module, I tried to re-install everythin but it is not working.
The rest of the app works fine as long as I don't use a component from another file, so the browser still shows "Hello World"
This is shown in the screenshot below, any help on how I can fix this so that I proceed.

Your help is highly appreciated.

![screenshot 2018-12-20 at 5 11 05 pm](https://user-images.githubusercontent.com/11255454/50292805-b7f02280-047a-11e9-973a-2931afd89644.png)

**Comment by benjamminj** [comment]
Hi! Sorry I just saw this, holidays and all kept me away from my computer. Hope you've figured it out since then, but still wanted to help out if not (:

Off the top of my head, could be related to your IDE, although the fact that the browser doesn't update means it could be something else. You might wanna try restarting the IDE and the build system.


That said, it's difficult to debug without something I can poke around. Do you have a branch on a repo that I can look at? That would be extremely helpful in figuring out what's going on

**Comment by OlivierJM** [comment]
Thanks very much @benjamminj 

I have been looking for a good entry resources on reasonml and your article is the greatest I found so far, I didn't manage to continue further than that.
I am going to push the code so that I can share it here, I did try to rebuild the project, my understanding was also that it had something to do with the text editor but it wasn't.

I will get back with the reproduction of that error.

Thank you

**Comment by OlivierJM** [comment]
I have pushed the repo, the code is here, I will appreciate for your help

https://github.com/OlivierJM/repo-reason

**Comment by benjamminj** [comment]
Strange, I'm still not seeing it, even when I pull down the code and run it. For reference, here's the commands I ran.

```bash
git clone https://github.com/OlivierJM/repo-reason.git
cd repo-reason
npm install
npm run dev
```

Given that it's working with a fresh install, I wonder if it's related to the built assets or something. You might want to try deleting `lib`, `dist`, `.cache`, and run a fresh build, although it sounds like you already tried that.

If that doesn't work and you're still getting errors, I'd check your IDE plugin. Some of the reason language plugins have issues ([This is the one I've been using](https://marketplace.visualstudio.com/items?itemName=jaredly.reason-vscode) and it works pretty well so far). Best way to check this is to look and see if your IDE is throwing errors but there's no errors coming from the `npm run dev` script in the console (and the browser displays the correct UI).

If neither of those work, shoot me an email directly—I think I've got a little time in the next week or so and we can set up a short video chat or something. I'm more than happy to help as much as I can.
